### ISSUE TITLE
Please make the Good Grief screenshots opt in again pretty please

### ISSUE THREAD

**Issue by Arcadius2006** [LittleBigRefresh/Refresh#696]
as expected, some people are unaware that Good Grief reports work this way, and people are trying to use it in the way the game intended the feature to be used. 

proof of this can be found here. https://beta.littlebigrefresh.com/photo/6802

i know good grief may be inferior to just talking about it on discord, but PLEASE make it optional, or atleast make good grief photos not immediately go to the public photo board

**Comment by Toastbrot236** [comment]
I agree, and I might work on this

**Comment by jvyden** [comment]
Agreed. As we shift to something more serious these will be necessary to handle properly.

This can't be done by a simple revert to the change to remove good grief in the first place; the internally stored structure will have to change.
